arm-servicebus/2014-09-01/swagger/servicebus.json

```diff
@@ -560,9 +560,343 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ServiceBus/namespaces/{namespaceName}/queues/{queueName}": {
+      "put": {
+        "tags": [
+          "Queues"
+        ],
+        "operationId": "Namespaces_CreateOrUpdateQueue",
```

Comment:
This would be hard to find for the developer. Why don't you name this as Queues_CreateOrUpdate.
The Queue creation will be in namespaces `client.namespaces.createOrUpdateQueues()`and all the other operations will be on Queues `client.queues.delete|get`. It would be better for discoverability if it was  `client.queues.createOrUpdate|delete|get`. It would be logical for a user to find things like that.

resourcegroup creation is a pre-requisite. That does not mean that all the operations are on `ResourceGroups_*`